Sage/carbon
# docs(switch): fix console errors which appear in dev environment

### Proposed behaviour

Address all the console errors that appear in the `Switch` component documentation.

### Current behaviour

Console errors appear for `Switch` component:
- tabIndex is `NaN`
- `labelSpacing` prop is a string, should be a number
- loading has type of string when a boolean is expected
- Each child in a list should have a unique key id

### Checklist




#### QA


### Additional context

The tabIndex error is addressed by PR #5212 

### Testing instructions



This can be tested by visiting the `switch -> test` story and `switch -> loading` story. Inspect console and their should be no errors apart from the tabIndex error message which is addressed in PR #5212
